src/main/java/salesforce/ui/pages/navbar/NavBarClassic.java

```diff
@@ -52,7 +52,7 @@
     /**
      * Clicks in Case menu button.
      */
-    private void clickAllMenuTab() {
+    public void clickAllMenuTab() {
```

Comment:
This method has to be private. You should create an other method to call clickAllMenuTab